src/main/java/org/opensearch/security/privileges/SecurityIndexAccessEvaluator.java

```diff
@@ -175,6 +196,21 @@ public PrivilegesEvaluatorResponse evaluate(
         return presponse;
     }
 
+    private boolean checkSystemIndexPermissionsForUser(ConfigModelV7.SecurityRoles securityRoles) {
+        Set<WildcardMatcher> userPermMatchers = securityRoles.getRoles()
```

Comment:
You can rename this variable to something like "UserPermissions" to avoid needing to add a comment.